src/PerformanceTests/Common.Data/Program.cs

```diff
@@ -32,9 +32,8 @@ static void Main(string[] args)
                 if (Environment.UserInteractive) Console.Title = PermutationParser.ToString(permutation);
 
                 var assembly = System.Reflection.Assembly.GetExecutingAssembly();
-                var tasks = permutation.Tests.Select(x => (IStartAndStop)assembly.CreateInstance(x)).ToArray();
-
-                Run(options, permutation, tasks);
+                var runnableTests = permutation.Tests.Select(x => (BaseRunner) assembly.CreateInstance(x)).ToList();
+                runnableTests.ForEach(s => s.Execute(permutation, options, endpointName));
```

Comment:
Performance tests should not run sequential nor parallel. Stability runs should have the ability to run multiple scenarios in parallel.